src/OrchardCore.Modules/OrchardCore.Localization/Assets/js/optionsEditor.js

```diff
@@ -0,0 +1,69 @@
+function initializeOptionsEditor(elem, data, defaultValue, selectedCulture) {
+
+    var previouslyChecked;
+
+    var store = {
```

Comment:
This can be removed entirely since we don't need to keep a state between 2 vue.js components.

Reply:
Can you do it?

Reply:
Ok doing it.